test/lit/passes/gufa-refs.wast

```diff
@@ -6069,8 +6069,13 @@
   ;; CHECK:      (func $test-set-bottom (type $2)
   ;; CHECK-NEXT:  (block ;; (replaces unreachable ArraySet we can't emit)
   ;; CHECK-NEXT:   (drop
-  ;; CHECK-NEXT:    (ref.cast nullref
-  ;; CHECK-NEXT:     (global.get $global)
+  ;; CHECK-NEXT:    (block (result nullref)
+  ;; CHECK-NEXT:     (drop
+  ;; CHECK-NEXT:      (ref.cast nullref
+  ;; CHECK-NEXT:       (global.get $global)
+  ;; CHECK-NEXT:      )
+  ;; CHECK-NEXT:     )
+  ;; CHECK-NEXT:     (ref.null none)
```

Comment:
Why did this change? Previously the ref.cast emitted a nullable bottom type, and it should still keep doing so IIUC... the description mentions a change to non-nullable ones, but that's not the case here, so it must be something else?

Reply:
This _is_ what I was referring to with non-nullable references because the possible contents in `$global` is a non-null value, even though its static type is nullable.

Reply:
Oh right, the actual value is non-nullable, thanks!